src/main/java/com/spring/app/services/mappers/EconomyLevelResultMapper.java

```diff
@@ -0,0 +1,50 @@
+package com.spring.app.services.mappers;
+
+import com.spring.app.DataNotFoundException;
+import com.spring.app.EconomicLevel;
+
+import java.util.List;
+
+/**
+ * Maps data set to single value and cast it to EconomicLevel
+ *
+ * @author Lyubov Ruzanova
+ */
+public class EconomyLevelResultMapper implements ResultMapper<List<Object>, EconomicLevel> {
```

Comment:
is this really needed?

Reply:
mmm... not really, I can relocate it as a simple method of the CountryProfileDTOResultMapper class